pyproject.toml

```diff
@@ -18,7 +18,7 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3 :: Only",
 ]
-keywords = ["ibapi", "tws", "asyncio", "jupyter", "interactive", "brokers", "async", "ib_insync"]
+keywords = ["ibapi", "tws", "asyncio", "jupyter", "interactive", "brokers", "async", "ib_async", "ib_insync"]
```

Comment:
I thought it would be appropriate to leave `ib_insync` here since this repository is built off of it and still shares many parallels to it, but I understand if this should be removed.  From a keyword perspective, including `ib_insync` might make it more clear that the two repositories are related.